macosx/Torrent.mm

```diff
@@ -276,6 +276,18 @@ - (void)startTransfer
     [self startTransferIgnoringQueue:NO];
 }
 
+- (void)startMagnetTransferAfterMetaDownload
+{
+    if ([self alertForRemainingDiskSpace])
```

Comment:
Seems like calling `[self alertForRemainingDiskSpace]` is problematic here; we don't actually know whether `Torrent.sizeLeft` is set at this point since we don't have any guarantee that the metainfo's been downloaded yet?

Reply:
Note: for my review, I didn't try the alert scenario.